src/gui/createtorrent.ui

```diff
@@ -239,19 +239,16 @@
      <property name="text">
       <string>Start seeding after creation</string>
      </property>
+     <property name="checked">
+      <bool>true</bool>
+     </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="checkIgnoreShareLimits">
-     <property name="enabled">
-      <bool>false</bool>
-     </property>
      <property name="text">
       <string>Ignore share ratio limits for this torrent</string>
      </property>
-     <property name="checked">
-      <bool>true</bool>
```

Comment:
@sledgehammer999 I just deleted my config and by default is **disabled**, just the same behavior that with the changes I made. If you want I can make it enabled by default but it's not the current behavior.